Assignment1/ho-taek_BESOPTAssignment/app/src/main/java/com/htk/sopt1/SignUpActivity.kt

```diff
@@ -0,0 +1,74 @@
+package com.htk.sopt1
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.widget.Toast
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContract
+import androidx.activity.result.contract.ActivityResultContracts
+import com.htk.sopt1.databinding.ActivitySignUpBinding
+import com.htk.sopt1.databinding.ActivitySigninBinding
+
+class SignUpActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySignUpBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignUpBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        SignupButtonClickEvent()
+    }
+
+
+    private fun SignupButtonClickEvent() {
+        binding.btRegister.setOnClickListener {
+            val name = binding.etName.text.toString()
+            val id = binding.etSignid.text.toString()
+            val password = binding.etSignpassword.text.toString()
+            if (name.isNullOrBlank() or id.isNullOrBlank() or password.isNullOrBlank()) {
+                //꼭 context 쓸때 액티비티이름 같이 적어주기!!!!
+                Toast.makeText(this@SignUpActivity, "빈 칸이 있는지 확인해주세요", Toast.LENGTH_SHORT)
+                        .show()
+            }else {
+                val intent = Intent()
+                intent.putExtra("id", id)
+                intent.putExtra("password", password)
```

Comment:
이것두 유정이 코드리뷰할때도 적었지만 ㅋㅋㅋㅋㅋ
```suggestion
                intent.putExtra("id", id)
                .putExtra("password", password)
```
이렇게 적는 방법도 있어용 :)

Reply:
오...이런 더 편한 방법이 감사합니다!!